targets/arduino_101/src/libc_support.c

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
Hi Sakari,

You were right about the libc_support.c. I didn’t need any of that to link. An update is on the pull request without any of the lib_support.c or the asm code.

Sergio

From: Sakari Poussa [mailto:notifications@github.com]
Sent: Monday, June 6, 2016 12:58 PM
To: Samsung/jerryscript <jerryscript@noreply.github.com>
Cc: Martinez Rodriguez, Sergio <sergio.martinez.rodriguez@intel.com>; Mention <mention@noreply.github.com>
Subject: Re: [Samsung/jerryscript] Wip/sergio/zephyr arduino101 (#1108)


In targets/arduino_101/src/libc_support.c<https://github.com/Samsung/jerryscript/pull/1108#discussion_r65878031>:

> @@ -0,0 +1,105 @@

> +/* Copyright 2016 Samsung Electronics Co., Ltd.

I still don't fully understand why this file (libc_support.c) is included. In the prj.conf you are using the newlib libc library which it not the Zephyr minimal nor JerryScript libc. The newlib comes from the compilers that Zephyr supports.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Samsung/jerryscript/pull/1108/files/c1b52476ac355f49ea39867feed4a1e08d43da10#r65878031>, or mute the thread<https://github.com/notifications/unsubscribe/AG5Kf8Acy6GqN4C8IgOYS25sRMJ6lseoks5qJAtkgaJpZM4IqhMW>.